### PR TITLE
[16.0][IMP]crm_timesheet: Make lead optionnal

### PR DESCRIPTION
**Pull request by flotho**
To reduce columns number in analytic lines

### CODE REVIEW

File: crm_timesheet/views/hr_timesheet_view.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
                     name="lead_id"
                     invisible="context.get('base_model_name') == 'crm.lead'"
                     groups="sales_team.group_sale_salesman"
+                    optional="hide"
```
Comment (pedrobaeza) [inline crm_timesheet/views/hr_timesheet_view.xml]: To keep the status quo:

```suggestion
                    optional="show"
```

Comment (flotho) [reply crm_timesheet/views/hr_timesheet_view.xml]: Thanks @pedrobaeza , 
You're right, statu quo is better.
Is there any persistence of the conf per user ?

Comment (pedrobaeza) [reply crm_timesheet/views/hr_timesheet_view.xml]: Odoo stores the shown/hidden columns by browser.

**Comment by pedrobaeza** [comment]
Please squash together both commits

**Comment by flotho** [comment]
ping @DorianMAG @JulienMartinez @GaelTorrecillas @GuiPRIME

**Review by pedrobaeza — APPROVED** [review]

**Review by christian-ramos-tecnativa — APPROVED** [review]

**Review by DorianMAG — APPROVED** [review]

**Pull request merged**